### ISSUE TITLE
'tuple' object is not callable

### ISSUE THREAD

**Issue by AddASecond** [zhenpeiyang/MVS2D#8]
Hi, there is a bug in your code that cause: 'tuple' object is not callable
training process is ok to run but always showing  'tuple' object is not callable

the log is:

Training
'tuple' object is not callable
'tuple' object is not callable
'tuple' object is not callable
/mypath/anaconda3/lib/python3.8/site-packages/torch/optim/lr_scheduler.py:129: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
/mypath/anaconda3/lib/python3.8/site-packages/torch/optim/lr_scheduler.py:154: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
'tuple' object is not callable
/mypath/anaconda3/lib/python3.8/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
'tuple' object is not callable
'tuple' object is not callable
'tuple' object is not callable

**Comment by zhenpeiyang** [comment]
Hi @AddASecond , which pytorch version are you using?

**Comment by AddASecond** [comment]
> Hi @AddASecond , which pytorch version are you using?

hi, I use 
torch '1.9.0+cu111'  
torchvision '0.10.0+cu111'
CUDA V11.1.105
NVIDIA driver 460.32.03

**Comment by zhenpeiyang** [comment]
Hi @AddASecond , we've only tested on torch==1.5.0, CUDA==10.1/10.2. Could you try these settings?
